packages/application-components/src/components/tab-header/tab.styles.ts

```diff
@@ -42,8 +43,8 @@ export const getLinkStyles = (
   isActive &&
     css`
       ${getBottomBorderStyles(uiKitDesignTokens.colorPrimary)}
-      & h4 {
-        color: ${uiKitDesignTokens.colorPrimary} !important;
+      & h3 {
```

Comment:
This was unused, has to be consistent with [the Heading `as` prop](https://github.com/commercetools/merchant-center-application-kit/blob/16de26adb0c77e580e5bf64de2ff0ad805672d7c/packages/application-components/src/components/tab-header/tab-header.tsx#L55)

Reply:
After the redesign, font color for the Headline (`color-solid`) has been matching the expected `<TabLabel>` font color.

In the new designs the TabLabel font color is `color-primary`, so we need to overwrite the default Headline font color.
<img width="535" alt="Screenshot 2024-02-23 at 13 51 05" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/ceaaa86a-38d6-4572-a4ed-3f07ceeac188">